ebusd/config.yaml

```diff
@@ -1,5 +1,5 @@
 name: eBUSd
-version: "0.82"
+version: "0.83"
```

Comment:
I am going to change version number to 0.82 so that we don't release anything until the docs have been updated

Reply:
That's fine - I went with 0 as major version until there are pre-built containers for this add-on wich appears to be best practice to save any SD Cards running the add-on from too many write ops during on-device compilation.

Other than that, Supervisor needs to have a new version number for config changes to appear, and maybe for some internal caches to expire

Reply:
OK, will change to 0.82 now so we can release when everything is ready for next version